NEWS

```diff
@@ -2,6 +2,25 @@
 
 Sandia OpenSHMEM NEWS -- history of user-visible changes.
 
+v1.4.5
+------
+- Added a complete OpenSHMEM teams API to the shmemx interfaces.
+- Added the OpenSHMEM wait/test vector API to the shmemx interfaces.
+- Added support for shared memory atomic operations with the appropriate
+  acquire/release memory ordering where required.  This feature requires
+  the --enable-shr-transport build flag.
+- Updated SOS to use the new OFI memory registration mode flags instead of
+  the deprecated FI_MR_SCALABLE/FI_MR_BASIC modes.
+- Updated the wait/test any/all/some "status" array semantics to reflect
+  recent changes to the OpenSHMEM specification.
+- Added a memory sync before returning from barriers to ensure shared memory
+  updates and remote updates cached in the NIC are visible in memory.
```

Comment:
Not sure if the sentence structure is correct for this one.  Should we change it to - 
` Added memory sync before returning from barriers to ensure shared memory
  updates and remote updates cached in the NIC that are visible in memory. `

Reply:
The "that" in the proposed change is throwing me off. What part needs correction in the original sentence?

Reply:
Oops. Sorry for that "that". That "that" was added by me trying to correct it. I am ok with the sentence as is. Sorry for the confusion.